apps/src/code-studio/pd/foorm/editor/form/FormEditorManager.jsx

```diff
@@ -218,14 +219,15 @@ class FormEditorManager extends React.Component {
           </a>{' '}
           to get started.
         </p>
-        <div>
-          <DropdownButton id="load_config" title="Load Form..." className="btn">
-            {this.getFormattedConfigurationDropdownOptions()}
-          </DropdownButton>
-          <Button onClick={this.initializeEmptyCodeMirror} className="btn">
-            New Form
-          </Button>
-        </div>
+        <FoormLoadButtons
+          resetCodeMirror={this.props.resetCodeMirror}
+          namesAndVersions={this.props.namesAndVersions}
+          setSelectedData={this.props.setFormData}
+          resetSelectedData={() => this.resetSelectedData()}
+          showCodeMirror={() => this.showCodeMirror()}
+          onSelect={this.loadConfiguration}
```

Comment:
would prefer this to be consistent with how you're handling the rest of your callbacks